site/design-notes/templated-strings.md

```diff
@@ -120,7 +120,7 @@ We can express template processing as _instance behavior_ on a policy object:
 String s = STR."Hello \{name}, I am \{age} years old.";
 ```
 
-where `STR` is a static instance of `TemplatePolicy` which captures the obvious
+where `STR` is a static field of `TemplatePolicy` which captures the obvious
```

Comment:
This change is not right; it is not a field of TemplatePolicy, but a statically imported field of _some class_, where the field implements TemplatePolicy.  Perhaps this is "static field whose type is `TEmplatePolicy`.